pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/SchedulerManager.java

```diff
@@ -71,7 +75,7 @@
  * The scheduler manager is used to compute scheduling of function instances
  * Only the leader computes new schedulings and writes assignments to the assignment topic
  * The lifecyle of this class is the following:
- *  1. When worker becomes leader, this class with me initialized
+ *  1. When worker becomes leader, this class with be initialized
```

Comment:
```suggestion
 *  1. When a worker becomes a leader, this class is initialized
```

Do you intend to mean this?